app/controllers/admin/tickets_controller.rb

```diff
@@ -34,22 +34,34 @@ def create
     end
 
     def show
-      @ticket = SupportTicket.find(params[:id])
+      ticket
```

Comment:
This might be better as a `before_action` for the `show` and `resolve` actions.

Reply:
resolved in https://github.com/rubyforgood/inkind-admin/pull/183